### ISSUE TITLE
Unexplained radiation leak

### ISSUE THREAD

**Issue by aaronmallen** [McJtyMods/DeepResonance#160]
I have a generator completely incased in dense obsidian and dense obsidian glass.  When right up against the glass I am experiencing radiation levels ~200 but if I step away from the generator setup ~3 block the radiation levels spike to 2000+ for 2-3 blocks and then drop back down to ~200.

```
Minecraft: 1.10.2
Pack: Direwolf20 1.3.0
Deep Resonance: 1.1x-1.4.2
```

**Comment by McJty** [comment]
yes, it is not always 100% accurate. It is kinda hard to compute this so not sure I can do a lot about that.
Note that levels under 10000 are really not that bad though

**Comment by Aulweas** [comment]
I am experiencing the same issue.  I am getting the debuff also when not standing directly up against the dense glass also.  Using an 84/100/100 crystal, standing next to the dense glass I get 450ish on the radiation meter.  If I take 2 steps back it jumps to 2+k.  If this is an error in how the meter is reading it, I shouldn't be also getting the negative effects should I?
